icu4c/source/i18n/unicode/measunit.h

```diff
@@ -72,159 +72,270 @@ enum UMeasureUnitComplexity {
 };
 
 /**
- * Enumeration for SI prefixes, such as "kilo".
+ * Enumeration for SI and binary prefixes, e.g. "kilo-", "nano-", "mebi".
  *
- * @draft ICU 67
+ * Enum values should be treated as opaque: use umeas_getPrefixPower() and
+ * umeas_getPrefixBase() to find their corresponding values.
+ *
+ * @draft ICU 69
  */
-typedef enum UMeasureSIPrefix {
+typedef enum UMeasurePrefix {
+    /**
+     * The absence of an SI or binary prefix.
+     *
+     * @draft ICU 69
+     */
+    UMEASURE_PREFIX_ONE = 30 + 0, // An arbitrary value
```

Comment:
Instead of using "30" as an offset for the SI prefixes. I prefer using a constant called for example `SI_PREFIX_OFFSET` which equals 30 or whatever value.

Then use this constant as an offset for all the other SI prefixes.

Reply:
I like limiting how much cruft is present in a public header file. Another constant means another exported symbol accessible to clients. This was discussed in the ICU-TC meeting, so reviewed and approved by the committee. I believe further changes  (to the public header file) would need another review.

Reply:
For public use we want people to use the new C functions umeas_getPrefixPower and umeas_getPrefixBase.  I don't feel strongly about whether to introduce `UPRV_SI_PREFIX_OFFSET` as an internal thing.  If we did, we could easily label it `@internal` to avoid an additional API review.

Reply:
For now, I'm leaving this as just UMEASURE_PREFIX_ONE - there's only one copy of "30", so it's really just a question of whether this constant needs an alias. An alias would make it clearer where the arbitrary value is used (here and umeas_getPrefixPower), but I think the number of references to UMEASURE_PREFIX_ONE is not too many.

I did look at how vscode handles documentation for the symbol, and decided thereby that the "arbitrary value" comment should rather be combined with the main documentation for this symbol: I went adhead and did this.